app/src/main/java/comcmput301f16t01/github/carrier/MockFareCalculator.java

```diff
@@ -11,10 +11,10 @@
      * @param end
      */
     public MockFareCalculator(Location start, Location end) {
-        super(start, end);
+        //super(start, end);
```

Comment:
I think I got confused actually because just before this I was reading the fixed version so I was like "why are we here again"? I think Kieter has a fixed version done (possibly in Master already?)